### ISSUE TITLE
[NBug] Could not find a part of the path 'C:\Users\buddina\AppData...

### ISSUE THREAD

**Issue by buddina** [gitextensions/gitextensions#7849]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\buddina\AppData\Roaming\GitExtensions\GitExtensions\GitExtensions.settings'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalCopy(String sourceFileName, String destFileName, Boolean overwrite, Boolean checkHost)
   at GitCommands.Settings.FileSettingsCache.SaveImpl()
   at GitCommands.SettingsCache.<>c__DisplayClass3_0.<LockedAction>b__0()
   at GitCommands.SettingsCache.LockedAction[T](Func`1 action)
   at GitCommands.Settings.SettingsContainer`2.Save()
   at GitUI.CommandsDialogs.FormSettings.Save()
   at GitUI.CommandsDialogs.SettingsDialog.Pages.ChecklistSettingsPage.SaveAndRescan_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Additional information
Please fix it

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.22.0.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3190.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Please update to the latest version

**Comment by gerhardol** [comment]
Duplicate of #7395
Fixed in #7398, in 3.3
